includes/class-api.php

```diff
@@ -267,6 +274,17 @@ public function sync_post( $post_args, $duplicate_action, $force_update = false
 			return $this->fix_term_relationships( $local_post['ID'], $post_args );
 		}
 
+		if ( ! empty( $this->ps_meta_repair_fields ) ) {
```

Comment:
What happens usually if this is empty? Can we reverse the condition to remove a level of nesting to make this block easier to read/